frontend/src/entities/group/detail/group-detail.tsx

```diff
@@ -47,6 +49,7 @@ export function GroupDetail () {
             dispatch(getGroup(groupName)).then((response) => {
```

Comment:
This isn't specific to this PR but looks like `getGroup` has the wrong return type defined. It is declared as returning `IGroup` but seems like it returns `{group: IGroup, permissions: Permission[]}`